src/options/options.js

```diff
@@ -20,3 +20,12 @@ RandomTips.init(tips).then(() => {
     RandomTips.setContext("options");
     RandomTips.showRandomTipIfWanted();
 });
+
+let dark= window.matchMedia('@media (prefers-color-scheme: dark)');
```

Comment:
Use editorconfig  -> that `=` style is a non-standard here. A space is missing. :smile: